src/Tizen.Multimedia.MediaPlayer/Player/PlayerEnums.cs

```diff
@@ -132,6 +132,8 @@ public enum PlayerState
         /// <summary>
         /// Preparation in progress.
         /// </summary>
+        /// <remarks>The Preparing state is translated to other states internally.
```

Comment:
After PrepareAsync(), The state is trnaslated to `Ready`, right? 
How about below?

`The Preparing state is translated to <see cref="PlayerState.Ready"/> state internally.`

Reply:
right. that looks better. i'll patch it again.